cores/esp32/Arduino.h

```diff
@@ -179,6 +179,11 @@ uint16_t makeWord(uint8_t h, uint8_t l);
 
 #define word(...) makeWord(__VA_ARGS__)
 
+#define ESP_LOOP_TASK_STACK_SIZE(sz) \
```

Comment:
How about we lose the `ESP_` prefix? Since it's not ESP's task, but rather Arduino's, I think we should stay clear of using `ESP_` as prefix for anything that is not ESP specific. @igrr any comment?

Reply:
The reason I have suggested this name was that it's Espressif-specific, as in, this macro won't work on any other Arduino core (Atmel, STM, etc).

I'm okay with other naming as long as it's clear that it's an extension we define, not something coming from a library or available on other cores.

Reply:
If we use more generic name, other cores that run or support RTOS could add the same functionality (it's not rare that we get asked to add something supported elsewhere).

Reply:
My suggestion is to call it 
```dart 
SET_LOOPTASK_STACK_SIZE(sz)
```

Reply:
I can get behind that if you split loop and task :D `SET_LOOP_TASK_STACK_SIZE(sz)`

Reply:
Done.